### DISCUSSION TITLE
During handling of the above exception, another exception occurred

### DISCUSSION THREAD

**Discussion by fricpto** [smartcontractkit/full-blockchain-solidity-course-py#1722] [Q&A] [answered]
when I debug my web3_py_simple_storage file I get a connection error and this message "During handling of the above exception, another exception occurred" on this line of code "nonce = w3.eth.getTransactionCount(my_address)"
<img width="641" alt="Capture" src="https://user-images.githubusercontent.com/100000316/192850230-83d5f6bb-0d24-4779-b6d8-352e77f88205.PNG">

**Comment by cromewar** [comment, marked as answer]
Hello @fricpto 
Well be sure you are not making any typo on the code, in this case the connection to RPC client.
You can try changing for `127.0.0.1` instead.
Cheers.

**Reply by fricpto** [reply]
I'm already connected with 127.0.0.1
